src/ol_infrastructure/applications/ocw_site/__main__.py

```diff
@@ -0,0 +1,80 @@
+from pulumi import Config, export
+from pulumi_aws import iam, s3
+from pulumi_vault import aws
+
+from ol_infrastructure.lib.aws.iam_helper import lint_iam_policy
+from ol_infrastructure.lib.ol_types import AWSBase
+from ol_infrastructure.lib.pulumi_helper import parse_stack
+
+ocw_site_config = Config("ocw_site")
+stack_info = parse_stack()
+aws_config = AWSBase(
+    tags={
+        "OU": "open-courseware",
+        "environment": f"applications_{stack_info.env_suffix}",
+    }
+)
+
+# Create S3 buckets
+# There are two buckets for each environment (QA, Production):
+# One for the that environment's draft site (where authors test content
+# changes), and one for the environment's live site.
+# See http://docs.odl.mit.edu/ocw-next/s3-buckets
+
+draft_bucket_name = f"ocw-content-draft-{stack_info.env_suffix}"
+live_bucket_name = f"ocw-content-live-{stack_info.env_suffix}"
+
+draft_bucket = s3.Bucket(
+    draft_bucket_name,
+    bucket=draft_bucket_name,
+    tags=aws_config.tags,
+)
+
+live_bucket = s3.Bucket(
+    live_bucket_name,
+    bucket=live_bucket_name,
+    tags=aws_config.tags,
+)
+
+policy_description = (
+    "Access controls for the CDN to be able to read from the"
+    f"{stack_info.env_suffix} website buckets"
+)
+s3_bucket_iam_policy = iam.Policy(
+    f"ocw-site-{stack_info.env_suffix}-policy",
+    description=policy_description,
+    path=f"/ol-applications/ocw-site/{stack_info.env_suffix}/",
+    name_prefix="aws-permissions-",
```

Comment:
This should be more descriptive to make it easy to visually identify its association with OCW course content buckets

Reply:
I was taking the ocw_studio role as an example for this. I'm not sure what the name_prefix signifies. Can you suggest something?

Reply:
That's the policy name that is created and is viewable in the IAM console. I would likely use something like`ocw-site-content-read-only-{stack_info.env_suffix}`

Reply:
OK. Is the one for OCW studio OK, then? Should we change it, too?  https://github.com/mitodl/ol-infrastructure/blob/d622b5572955943925d1b3dc1e90e381eae93815/src/ol_infrastructure/applications/ocw_studio/__main__.py#L61

Reply:
We should probably change that one as well, but we can tackle that later.

Reply:
Name changed in 85bf03b (for this project)